subset/bacnet/bacnetTests/.idea/gradle.xml

```diff
@@ -15,4 +15,4 @@
       </GradleProjectSettings>
     </option>
```

Comment:
@JoelRedstone I think these files are fairly redundant, see Trevor's cloud test:

https://github.com/grafnu/daq/tree/master/subset/cloud

And Ed's switch test:

https://github.com/arupiot/daq/tree/master/subset/switches

For what should be in the repo

Reply:
@JoelRedstone This is Trevor's pubber tool:

https://github.com/arupiot/daq/tree/master/pubber

...actually I could be off the mark about not including the `.idea` folder - maybe it includes important build stuff I don't know about?

Reply:
I'll follow the pubber folder as an example.